docker/scripts/setup

```diff
@@ -29,7 +29,7 @@ if [ -d "/scripts/cache" ]; then
   mv /scripts/cache vendor/
   chown -R huginn:huginn vendor/cache
 fi
-sudo -u huginn -H bundle install --deployment --without development test
+sudo -u huginn -H bundle install --deployment --with deployment
```

Comment:
I do, that's why I added the "not recommended for production" label to the docs. However since the docker image has been running in development for ever I think we should fix it first and then figure out how to run it in production.